src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/token/PasswordResetToken.java

```diff
@@ -0,0 +1,82 @@
+package de.terrestris.shogun2.model.token;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/**
+ *
+ * The PasswordResetToken class.
+ *
+ * @author Daniel Koch
+ *
+ */
+@Entity
+@Table
+public class PasswordResetToken extends Token {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * The password token should be valid for 48h.
+	 */
+	private static final int expiration = 48;
```

Comment:
Yes. I'm a bit confused as we have two static variables with different values!??

`private static final Token.expiration` and `private static final PasswordResetToken.expiration` ?